src/TokenManagers/MinecraftBedrockTokenManager.js

```diff
@@ -10,7 +10,7 @@ class BedrockTokenManager {
   constructor (cacheLocation) {
     this.cacheLocation = cacheLocation || path.join(__dirname, './bed-cache.json')
     try {
-      this.cache = require(this.cacheLocation)
```

Comment:
You could try to get the absolute path to the cache file using path.resolve instead of path.join, that should do it.

Reply:
Yes, but using require has other side effects like caching which is not desirable, and the cause of obscure bugs. This also brings parity with how we write to the cache.